packages/slas-react-sdk/src/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { createContext, useContext } from "react";
+import { createStore, useStore } from "zustand";
+import { persist } from "zustand/middleware";
+import { helpers, ShopperLogin } from "commerce-sdk-isomorphic";
+import { onClient, getAppOrigin } from "./util";
+import { isExpired } from "./jwt";
+import { MemoryStorage } from "./storage";
+
+enum AuthTypes {
+  REGISTERED = "registered",
+  GUEST = "guest",
+}
+
+interface State {
+  isInitialized: boolean;
+  isInitializing: boolean;
+  isAuthenticated: boolean;
+  authType: AuthTypes;
+  customerId: string;
+  accessToken: string;
+  refreshToken: string;
+  _config: CommerceAPIConfig;
+  login: (username: string, password: string) => void;
+  register: (username: string, password: string) => void;
+  loginAsGuest: () => void;
+  logout: () => void;
+  _initialize: () => void;
+  _refreshAccessToken: () => void;
+  _handleShopperLoginTokenResponse: (res: helpers.TokenResponse) => void;
```

Comment:
This object's type seems to be a combination of TokenResponse, ShopperLoginParameters (config) and some other things. Maybe we can clean this up by grouping some of these properties a little better.

Also, is it required to have these private functions in the state?